zulipterminal/model.py

```diff
@@ -945,24 +947,44 @@ def get_stream_by_id(streams: List[StreamData], stream_id: int
                         for user_id in user_ids:
                             subscribers.remove(user_id)
 
+    @asynch
+    def _show_typing_animation(self, recipient: Dict[str, Any]) -> None:
+        start_time = datetime.now()
+        # Restore the footer if there hasn't been a corresponding typing
+        # 'stop' event after a 'start' event for greater than 15 seconds.
+        max_duration = timedelta(seconds=TYPING_STARTED_EXPIRY_PERIOD)
```

Comment:
Has this constant (15 seconds) been borrowed from somewhere?

Reply:
Yes this is the same constant used in @prah23's #915 where I have borrowed it from and also the same is recommended in the API docs for typing events given [here](https://zulip.com/api/set-typing-status).